native/cocos/network/HttpClient-java.cpp

```diff
@@ -833,6 +833,7 @@ HttpClient::HttpClient()
   _cookie(nullptr),
   _requestSentinel(ccnew HttpRequest()) {
     CC_LOG_DEBUG("In the constructor of HttpClient!");
+    _requestSentinel->addRef();
```

Comment:
From 3.6, Ref class 's default reference count is 0 instead of 1 now. So need to `addRef` after new.